drivers/syslog/Kconfig

```diff
@@ -273,23 +273,26 @@ menuconfig SYSLOG_FILE
 
 if SYSLOG_FILE
 
-config SYSLOG_FILE_ROTATE
-	bool "Log file rotate"
-	default n
+config SYSLOG_FILE_ROTATIONS
+	int "Log file rotations"
+	default 0
 	depends on SYSLOG_FILE
 	---help---
-		If enabled, the log file size will be checked before opening.
-		If it is larger than the specified limit it will be "rotated",
-		i.e. the old file will be kept as a backup, and a new empty
-		file will be created.
+		If enabled (set to a non-zero number), the log file size will be
+		checked before opening. If it is larger than the specified limit
+		it will be "rotated", i.e. the old file will be kept as a backup,
+		and a new empty	file will be created.
+		
```

Comment:
It is not common to add empty lines on the help text of Kconfig options.
Maybe you should remove them.

Reply:
I think I have seen it in many places across NuttX.

I can change this. But in general, how do you suggest to separate "paragraphs"? Or just leave all the text in a single chunk?

Reply:
Sorry, you may ignore this, I've just spent some minutes gathering more NuttX Kconfig samples and there are several places that add empty lines.